### PR TITLE
Flickable off

### PR DESCRIPTION
**Pull request by vladest**
Temporarily switch off flickable but keep infrastructure for future R&D
Fixes the following:

https://highfidelity.fogbugz.com/f/cases/7496/User-agent-doesn-t-allow-you-to-download
https://highfidelity.fogbugz.com/f/cases/7495/Flickable-Webviews-Don-t-Resize-Page-Correctly
https://highfidelity.fogbugz.com/f/cases/7546/Marketplace-Input-Field-Does-not-Work
https://highfidelity.fogbugz.com/f/filters/177/My-Cases

Test plan:
- Open Marketplace/Blocks/Create/Location infos/etc
- make sure everything works as before, except flicking

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3218/HighFidelity-Beta-PR11341-08deba35ae2bccc578765d734fbdc590f65dd022.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3218/HighFidelity-Beta-PR11341-08deba35ae2bccc578765d734fbdc590f65dd022.exe

File: interface/resources/qml/controls/FlickableWebViewCore.qml [hunk 1]
```diff
@@ -6,54 +6,33 @@ import QtQuick.Controls 2.2
 
 import "../styles-uit" as StylesUIt
 
-Flickable {
+Item {
     id: flick
 
     property alias url: _webview.url
```
Comment (druiz17) [inline interface/resources/qml/controls/FlickableWebViewCore.qml]: vlad I missed this last time, but qml files don't use the pattern of putting underscores at the beginning of variable names. Can you remove them from this file.

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3226/HighFidelity-Beta-PR11341-92a83328c00cb90ee1ab24f6bd12f325a8055a92.exe

**Comment by themelissabrown** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3232/HighFidelity-Beta-PR11341-92a83328c00cb90ee1ab24f6bd12f325a8055a92.exe

**Comment by marko8904** [comment]
built this please

**Comment by marko8904** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3236/HighFidelity-Beta-PR11341-92a83328c00cb90ee1ab24f6bd12f325a8055a92.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3236/HighFidelity-Beta-PR11341-92a83328c00cb90ee1ab24f6bd12f325a8055a92.exe

**Pull request merged**